pkg/artifacts/artifacts_storage.go

```diff
@@ -68,11 +54,31 @@ func InitializeArtifactStorage(pr *v1alpha1.PipelineRun, c kubernetes.Interface)
 	return NewArtifactBucketConfigFromConfigMap(configMap)
 }
 
+func needsPVC(configMap *corev1.ConfigMap, err error) bool {
+	if err != nil {
+		return true
```

Comment:
hm in this error case we should definitely at least log i think (unless it's a not found error, which we can probably check for explicitly) - if not even fail the run?

Reply:
good point :+1: didn't change the current behavior but if it's not a *not found* error, it seems fishy and we may fail the run as a `fail early` :angel: